src/modules/contract/contract.controller.ts

```
@@ -11,10 +12,20 @@
+  @ApiConsumes('multipart/form-data')
+  @ApiBody({
+    schema: {
+      type: 'object',
+      properties: {
+        file: { type: 'string', format: 'binary' },
+        contractType: { type: 'string' },
+      },
+    },
+  })
+  @UseInterceptors(FileInterceptor('file'))
+  create(@UploadedFile() file: Express.Multer.File, @Body('contractType') contractType: string) {
```

ESLint

[error] Replace `@UploadedFile()·file:·Express.Multer.File,·@Body('contractType')·contractType:·string` with `⏎····@UploadedFile()·file:·Express.Multer.File,⏎····@Body('contractType')·contractType:·string,⏎··`

(prettier/prettier)